Include/internal/pycore_runtime_init.h

```diff
@@ -93,26 +93,34 @@ extern "C" {
         _PyBytes_SIMPLE_INIT(CH, 1) \
     }
 
-#define _PyASCIIObject_INIT(LITERAL) \
+#define _PyASCIIObjectBase_INIT(LITERAL, ASCII) \
```

Comment:
My comment suggesting `_PyUnicode_LATIN1_BASE_INIT` made me realize it might be worth clarifying this name too:

```suggestion
#define _PyUnicode_ASCII_BASE_INIT(LITERAL, ASCII) \
```

Reply:
FIxed